sim/side.js

```diff
@@ -470,18 +470,17 @@ class Side {
 		if (index >= this.active.length) {
 			return this.emitChoiceError(`Can't switch: You do not have a Pokémon in slot ${index + 1}`);
 		}
-		/**@type {Pokemon} */
-		// @ts-ignore
 		const pokemon = this.active[index];
-		const autoChoose = !slotText;
-		let slot = parseInt(slotText) - 1;
-		if (autoChoose) {
+		let slot;
+		if (!slotText) {
```

Comment:
The code is more readable if we keep ``autoChoose`` I think?
The variable name signals intention; otherwise, one would need to add a comment saying `` // default to automatically choosing`` or something alike.

Reply:
Fair enough. I originally changed it because TypeScript didn't recognize <code>slotText</code> to be defined in the <code>else</code> block.